bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -33,7 +33,7 @@ import scalaz.effect.IO
 import scalaz.effect.IO.ioUnit
 
 /** The state of AGS lookup for a single observation. */
-sealed trait BagsState {
+sealed class BagsState {
```

Comment:
Agreed. I already changed it to sealed abstract class, but it should be sealed trait, I suppose.